trade-finance-manager-ui/server/controllers/utilisation-reports/record-corrections/create-record-correction-request/form-helpers.ts

```diff
@@ -0,0 +1,50 @@
+import { isNonEmptyString, RECORD_CORRECTION_REASON, RecordCorrectionReason } from '@ukef/dtfs2-common';
+import { CreateRecordCorrectionRequestFormValues } from '../../../../types/view-models';
+
+/**
+ * Values for the record correction reasons as a string array.
+ */
+const RECORD_CORRECTION_REASON_VALUES = Object.values(RECORD_CORRECTION_REASON) as string[];
```

Comment:
Can we address the Codacy issue 


```
This assertion is unnecessary since it does not change the type of the expression.
```

Reply:
Tried this originally but this results in a TS error further down where we have the type guard:

![Error screenshot](https://github.com/user-attachments/assets/a755e6c5-efdd-4b9d-9477-cbfbdbbc8a76)